### DISCUSSION TITLE
個人建議...關於Quiz

### DISCUSSION THREAD

**Discussion by Rebecca111419** [1101-GS4719-C/Python-Community-GS4719-C#1698] [General]
建議Quiz 答對後不要自動跳下一題，因為有時想再看一下題目與正確答案，對照詳解(特別是矇到的時候
不知是否有人和我一樣的小困擾，答完需要再按回題目

**Comment by chia-kai-chang** [comment]
謝謝 @Rebecca111419 的建議，我們來修改一下程式。修改後再請 @Rebecca111419 測試一下喔，如果有要調整的地方在跟我們說或是在這邊反映。

**Reply by Rebecca111419** [reply]
哦好~ 謝謝老師~~
